.github/workflows/create-release-pull-request.yaml

```diff
@@ -21,10 +21,18 @@ jobs:
 
       - name: Start Changelog entry
         run: |
-          function dt {
-            date +'%Y-%m-%d'
-          }
-          sed -i "s/<!--Start Of Releases (DO NOT DELETE THIS LINE)-->/<!--Start Of Releases (DO NOT DELETE THIS LINE)-->\n\n## [${{ github.event.inputs.newVersionNumber }}] - $(date +%F)\n\n## Highlights\n\n## What’s Changed\n\n## New Contributors/" ./CHANGELOG.md
+          echo ${{ secrets.GITHUB_TOKEN }} | gh auth login --with-token
```

Comment:
`echo`ing secrets always scares me (thought GitHub is pretty good at masking them).

This https://docs.github.com/en/actions/using-workflows/using-github-cli-in-workflows seems to suggest you can use syntax like this instead:

```
jobs:
  comment:
    runs-on: ubuntu-latest
    steps:
      - run: gh issue comment $ISSUE --body "Thank you for opening this issue!"
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          ISSUE: ${{ github.event.issue.html_url }}
```

Reply:
FWIW, it did mask, but yeah, I will try this out.

<img width="565" alt="Screen Shot 2022-05-17 at 11 48 39 PM" src="https://user-images.githubusercontent.com/28961086/168975438-86e41e3c-42fc-4535-a1d6-a4dad43bc306.png">

Reply:
OK, this worked on my end: https://github.com/sqlfluff/sqlfluff/pull/3360/commits/bdca2a05a25a5dd37b1b06483a4e2093199b813f